MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -361,8 +365,8 @@ internal void PacketReader(object? o)
         /// <returns>TRUE if the packet was processed, FALSE if ignored or unknown</returns>
         internal bool HandlePacket(int packetId, Queue<byte> packetData)
         {
-            try
-            {
+            //try
+            //{
```

Comment:
I forgot to uncomment the try-catch here. Remember to uncomment it before release.

Reply:
Done